README.md

```diff
@@ -17,9 +48,11 @@ end
 If using VS Code, install the [Ruby LSP plugin](https://github.com/Shopify/vscode-ruby-lsp) to get the extra features in
 the editor.
 
-See the [documentation](https://shopify.github.io/ruby-lsp) for
+See the [documentation](https://shopify.github.io/ruby-lsp) for more in-depth details about the
 [supported features](https://shopify.github.io/ruby-lsp/RubyLsp/Requests.html).
 
+Currently is not possible to use a 'globally installed' version of the gem - it must be part of the `Gemfile`.
```

Comment:
You can actually use it from a global installation for editors other than VS Code (e.g.: NeoVim) as long as you're not using RuboCop.